File: 프로그래머스/lv3/장정윤/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    for char in s:
        cnt = 0
        curr_char = char
        
        while cnt < index:
            curr_char = chr(ord(curr_char)+1)
            if curr_char > 'z':
                curr_char = chr(ord(curr_char) - 26)
            if curr_char not in skip:
                cnt += 1
        answer += curr_char   
    return answer

File: 프로그래머스/lv2/이선영/둘만의 암호.java
import java.util.*;

class Solution {
    public String solution(String s, String skip, int index) {
        String answer = "";
        StringBuilder sb = new StringBuilder();
        for(int i=0; i<s.length(); i++){
            char c = s.charAt(i);
            int cnt = 0;
            while(cnt < index){
                c++;
                if(c > 'z') c='a';
                if(!skip.contains(Character.toString(c))){
                    cnt++;
                }
            }
            
            sb.append(c);
           
        }
        answer = sb.toString();
        
        return answer;
    }
}
